README.md

```diff
@@ -1 +1,76 @@
 # dsaps
+
+This command line application provides several ways of interacting with the [DSpace](https://github.com/DSpace/DSpace) API. This application was written for DSpace 6.3, it has not been tested against other DSpace versions.
+
+## Installation
+Clone the repository and install using [pipenv](https://github.com/pypa/pipenv):
+```
+pipenv install
+```
+After installation, run the application with:
+```
+pipenv run dsaps
+```
+
+## Authentication
+
+To authenticate, use the following parameters
+
+Option (short) | Option (long)     | Description
+------ | ------ | -----------
+N/A | --url | The DSpace API URL (e.g. https://dspace.mit.edu/rest), defaults to the DSPACE_URL environmental variable if nothing is specified
+-e | --email | The email of the user for authentication.
+-p | --password | The password for authentication.
+
+## Commands
+
+### additems
+Adds items to a specified collection from a metadata CSV, a field mapping file, and a directory of files. May be run in conjunction with the newcollection CLI command.
+
+Option (short) | Option (long)             | Description
+------ | ------ | -------
+-m | --metadata-csv | The path to the CSV file of metadata for the items.
+-f | --field-map | The path to JSON field mapping file.
+-d | --content-directory | The full path to the content, either a directory of files or a URL for the storage location.
+-t | --file-type | The file type to be uploaded, if limited to one file type.
+-r | --ingest-report| Create ingest report for updating other systems.
+-c | --collection-handle | The handle of the collection to which items are being added.
+
+
+#### Example Usage
+```
+dsaps --url https://dspace.com/rest -e abc@def.com additems -m coll_metadata.csv
```

Comment:
A few notes for all three of these example usage blocks:

- Include `pipenv run ` at the beginning since that's likely how someone would run it manually
- I would include the password option since it's required
- Either remove the new lines and let these commands blocks soft wrap or put a blackslash immediately before each new line, otherwise when copy/pasted into a shell it will read each line as a new command